CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ find_package(FLEX REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 
 ExternalProject_Add(rcssserver
-  GIT_REPOSITORY "https://github.com/mhauskn/rcssserver.git"
+  GIT_REPOSITORY "https://github.com/CoolPhilChen/rcssserver.git"
```

Comment:
Were there changes you made to the rcssserver?

Reply:
No. All the changes I made are here https://github.com/LARG/HFO/commit/7db4f8025e141bc634095509a1e7e45732840e3d

Reply:
I’ll make a new pull request.